ProjectSettings/ProjectVersion.txt

```diff
@@ -1 +1 @@
-m_EditorVersion: 5.6.2f1
+m_EditorVersion: 5.6.0f3
```

Comment:
I have write access to repo I forked from this, but the changes requested from here don't show up there I guess?